api/src/main/java/org/openmrs/Concept.java

```diff
@@ -641,20 +641,20 @@ private ConceptName getNameInLocale(Locale locale) {
 	 * Returns the name which is explicitly marked as preferred for a given locale.
 	 * 
 	 * @param forLocale locale for which to return a preferred name
+	 * @param  
 	 * @return preferred name for the locale, or null if no preferred name is specified
 	 * @should return the concept name explicitly marked as locale preferred
 	 * @should return the fully specified name if no name is explicitly marked as locale preferred
 	 */
-	public ConceptName getPreferredName(Locale forLocale) {
+	public ConceptName getPreferredName(Locale forLocale, object ) {
```

Comment:
I don't see this parameter used anywhere so my suggestion is to remove this and also seems this is the reason behind the build failure since the object param does not have a type.